lib/components/Text/Paragraph.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { FC, HTMLAttributes } from "react";
+import * as styles from "./styles";
+
+type Props = HTMLAttributes<HTMLParagraphElement>;
+
+export const Paragraph: FC<Props> = (props) => {
+  return (
+    <p className={styles.copy} {...props}>
```

Comment:
probably better to merge classnames rather than completely override. See my other comment on the `<H2>`